podspec_constants.rb

```diff
@@ -1,5 +1,5 @@
 module SquareInAppPaymentsSDK
-  VERSION = '1.4.0'
+  VERSION = '1.5.0'
```

Comment:
there is also another file. Verticals/in-app-payments-sdk/Scripts/Pod/constants.rb which stores versions